abr_control/controllers/signals/friction.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+
+from .signal import Signal
+
+class Friction(Signal):
```

Comment:
fake friction signal, should this be in the dewolf repo?

Reply:
yup